clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -194,9 +195,13 @@ class ClioWebClient(
       Uri.Path(s"/api/v1/${clioIndex.urlSegment}/metadata")
     )(_ / _)
 
+    val uri =
+      if (force) Uri(path = encodedPath).withQuery(Uri.Query("force" -> "true"))
+      else Uri(path = encodedPath)
```

Comment:
any way to avoid calling `Uri()` constructor on two different code paths here? e.g. something like `Uri(path = encodedPath).map(if (force) _.withQuery(...) else _)`